catalogue/webapp/pages/item.js

```diff
@@ -243,22 +243,29 @@ const PaginatorButtons = ({
       })}
     >
       {prevLink && (
-        <NextLink {...prevLink} scroll={false} replace>
-          <a
-            className={classNames({
-              [spacing({ s: 1 }, { margin: ['right'] })]: true,
-            })}
-          >
-            <Control type="light" icon="arrow" extraClasses="icon--180" />
-          </a>
-        </NextLink>
+        <Control
+          scroll={false}
+          replace={true}
+          nextLink={prevLink}
```

Comment:
I think we've just called it link elsewhere.

Reply:
the prop, or the component? Or both?

Reply:
The `prop`.